### ISSUE TITLE
Feature Request/Idea: Improve alt text for image files in datasets

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse#8370]
**Overview of the Feature Request**
When a dataset's image file is displayed, it might improve accessibility if its alt text is more descriptive.

As of Dataverse v5.9, image files in datasets are displayed as thumbnails on several pages, including in the search cards on search pages and in the "Files" tab on dataset pages, and the alt text for those image displays seems to be either the dataset title or the name of the image file, depending on where the thumbnail appears.

When the image is displayed on the File page in the "Metadata" tab, the alt text is the name of the image file. When it's displayed on the "Preview" tab by the Previewers tool, the alt text is an empty string.

**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
Curator, Depositor, Guest

**What inspired the request?**
A depositor recently emailed a repository to ask how they could include a description of their images that can be used for the alt text.

**What existing behavior do you want changed?**
When the images in datasets are displayed, e.g. in thumbnails and on the file pages, the value for the alt text attribute can be something that the dataset depositor or a curator can define instead of using the file's name or the dataset title.

**Any brand new behavior do you want to add to Dataverse?**
Allowing the depositor or curator to define the alt text might be done by adding a field at the file level for entering alt text or using what's entered in the file's Description field.

**Any related open or closed issues to this feature request?**

**Comment by DS-INRAE** [comment]
Even though this is still a very important topic, we will release it at t should be addressed in the SPA rather than the JSF
